test/e2e/core.go

```diff
@@ -97,3 +110,29 @@ func logCore(cmds []command, hosts []string, dir, provider string) {
 	}
 	wg.Wait()
 }
+
+func isUsingSystemdKubelet(provider string, hosts ...string) bool {
+	wg := &sync.WaitGroup{}
+	results := make([]int, len(hosts))
+	cmd := "sudo systemctl status kubelet.service"
+
+	wg.Add(len(hosts))
+	for i := range hosts {
+		go func(i int) {
+			defer wg.Done()
+			result, err := SSH(cmd, hosts[i], provider)
+			if err != nil {
+				fmt.Printf("Error running command: %v\n", err)
+			}
+			results[i] = result.Code
```

Comment:
@jonboulle So in this case we assume the kubelet is running as systemd service even when there are errors during SSH().